opflexagent/apic_topology.py

```diff
@@ -146,11 +145,14 @@ def _check_for_new_peers(self, context):
                     LOG.debug('reporting peer removal: %s', peer)
                     self.service_agent.update_link(
                         context, peer[0], peer[1], None, 0, 0, 0, 0, '')
+                    self.count_current = 0
                 if (interface not in curr_peers or
                         curr_peers[interface] != peer or
                         force_send):
                     LOG.debug('reporting new peer: %s', peer)
-                    self.service_agent.update_link(context, *peer)
+                    self.service_agent.update_link(context, *peer,
+                                                   force=force_send)
+                    self.count_current = 0
                 if interface in curr_peers:
                     curr_peers.pop(interface)
 
```

Comment:
do we have to set count_current to 0 also after the update_link() call at line#163? I saw its getting udpated at line#148 so.